catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/workspace/QueryMetacardImpl.java

```diff
@@ -37,6 +39,16 @@ public QueryMetacardImpl(String title) {
   public QueryMetacardImpl(Metacard wrappedMetacard) {
     super(wrappedMetacard, TYPE);
     setTags(Collections.singleton(QueryMetacardTypeImpl.QUERY_TAG));
+
+    for (AttributeDescriptor attrDesc : TYPE.getAttributeDescriptors()) {
```

Comment:
The constructor called by `super` already copies all the attributes to this metacard.